sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/Tests/DnsResolverTests.cs

```diff
@@ -8,15 +8,14 @@
 using Azure.ResourceManager.Resources.Models;
 using Azure.Core;
 using Azure.ResourceManager.DnsResolver.Models;
-using System.Linq;
 
 namespace Azure.ResourceManager.DnsResolver.Tests
 {
     public class DnsResolverTests : DnsResolverTestBase
     {
         private DnsResolverCollection dnsResolverCollection;
 
-        public DnsResolverTests(bool async) : base(async)
+        public DnsResolverTests(bool async) : base(async, RecordedTestMode.Playback)
```

Comment:
We will run the test in CI with playback mode, therefore you don't need to add the parameter in the ctor of base class.
```suggestion
        public DnsResolverTests(bool async) : base(async)//, RecordedTestMode.Record)
```